packages/salesforcedx-vscode-core/package.json

```diff
@@ -38,7 +39,7 @@
     "@types/chai": "^4.0.0",
     "@types/glob": "^5.0.30",
     "@types/mocha": "2.2.38",
-    "@types/node": "^7.0.0",
+    "@types/node": "^8.0.0",
```

Comment:
@salesforce/core wouldn't compile without updating @types/node

Reply:
VS Code just updated to the latest electron which contains Node 8. So I think we can change this.

Reply:
@lcampos - Could we upgrade all the @types/node to 8.00? Also, let's not use the ^ since that has caused us issues in the past when someone changed the type in a non-backwards compatible way and caused compile to fail.

Reply:
@vazexqi sure, I'll create a separate PR for that.